Objects/setobject.c

```diff
@@ -2014,6 +2014,29 @@ set_init(PySetObject *self, PyObject *args, PyObject *kwds)
     return set_update_internal(self, iterable);
 }
 
+static PyObject*
+set_vectorcall(PyObject *type, PyObject * const*args,
+               size_t nargsf, PyObject *kwnames)
+{
+    if (kwnames && PyTuple_GET_SIZE(kwnames) != 0) {
```

Comment:
I will replace it into `_PyArg_NoKwnames`
When https://github.com/python/cpython/pull/18980 is merged.

Reply:
Yeah, we can do that later.